web/themes/joinup/prototype/sass/layouts/_layout.sass

```diff
@@ -10,27 +10,12 @@ div
 div
   box-sizing: border-box
 
-.main-container,
-.footer-container
-  box-sizing: border-box
+.main-container
   width: 100%
+  padding-left: 16px
```

Comment:
Hm, so far I was looking at the `row` classes clearing padding, and here the padding is added. I don't really understand the approach.